src/main/webapp/index.html

```diff
@@ -5,13 +5,15 @@
   <meta charset="UTF-8">
   <title>Graphs</title>
   <link rel="stylesheet" href="style.css">
-  <script src="script.js"></script>
+  <script src="cytoscape/cytoscape.min.js"></script>
```

Comment:
Since you are only using `min.js` do you need to check in the other versions `.umd.js`, `.esm.js`, `.cjs.js`?

Reply:
Instead of checking this in, we can directly include it from one of the CDNs.

See https://js.cytoscape.org/#getting-started/including-cytoscape.js

Reply:
In a different branch I also added package management using npm since a lot of the cytoscape plugins are included that way. Do you think importing cytoscape that way would be okay?

Reply:
OH i should've said this before resolving, but I used https://cdnjs.cloudflare.com/ajax/libs/cytoscape/3.15.1/cytoscape.min.js since it said it was the default. Would this be compatible with Ishani's?

Reply:
Cindy's link should be the same file as the previously check-in version so I think this is fine!